src/legacy/legacy-grpc-network-client.ts

```diff
@@ -1,22 +1,22 @@
-import * as Networking from "./network-client";
-import { XRPLedgerAPIClient } from "../generated/legacy/xrp_ledger_grpc_pb";
-import { AccountInfo } from "../generated/legacy/account_info_pb";
-import { Fee } from "../generated/legacy/fee_pb";
-import { GetFeeRequest } from "../generated/legacy/get_fee_request_pb";
-import { GetAccountInfoRequest } from "../generated/legacy/get_account_info_request_pb";
-import { SubmitSignedTransactionRequest } from "../generated/legacy/submit_signed_transaction_request_pb";
-import { SubmitSignedTransactionResponse } from "../generated/legacy/submit_signed_transaction_response_pb";
-import { GetLatestValidatedLedgerSequenceRequest } from "../generated/legacy/get_latest_validated_ledger_sequence_request_pb";
-import { LedgerSequence } from "../generated/legacy/ledger_sequence_pb";
-import { GetTransactionStatusRequest } from "../generated/legacy/get_transaction_status_request_pb";
-import { TransactionStatus } from "../generated/legacy/transaction_status_pb";
+import { XRPLedgerAPIClient } from "../../generated/legacy/xrp_ledger_grpc_pb";
+import { AccountInfo } from "../../generated/legacy/account_info_pb";
+import { Fee } from "../../generated/legacy/fee_pb";
+import { GetFeeRequest } from "../../generated/legacy/get_fee_request_pb";
+import { GetAccountInfoRequest } from "../../generated/legacy/get_account_info_request_pb";
+import { SubmitSignedTransactionRequest } from "../../generated/legacy/submit_signed_transaction_request_pb";
+import { SubmitSignedTransactionResponse } from "../../generated/legacy/submit_signed_transaction_response_pb";
+import { GetLatestValidatedLedgerSequenceRequest } from "../../generated/legacy/get_latest_validated_ledger_sequence_request_pb";
+import { LedgerSequence } from "../../generated/legacy/ledger_sequence_pb";
+import { GetTransactionStatusRequest } from "../../generated/legacy/get_transaction_status_request_pb";
+import { TransactionStatus } from "../../generated/legacy/transaction_status_pb";
+import { LegacyNetworkClient } from "./legacy-network-client";
```

Comment:
Any specific reason you changed the import from `* as Networking` to `{ LegacyNetworkClient }`?

Reply:
No. I think this was actually just a bad VSCode autocomplete at some point. Any reason you'd like me to namespace this object?

Reply:
I can't think of any, it was just a change that stood out to me as different from the other `import` changes.